src/test/java/org/llorllale/cactoos/matchers/HasEntryTest.java

```diff
@@ -75,9 +75,9 @@ public void doesNotMatchAMissingEntry() {
 
     @Test(expected = AssertionError.class)
```

Comment:
need to get rid of Junit4-style assertions and use `Throws`

Reply:
@ilyakharlamov doing that reduces the coverage which means this PR will be blocked. There is a puzzle open for this at the top of this file. See https://github.com/llorllale/cactoos-matchers/pull/105#issuecomment-486203698